main.go

```diff
@@ -730,12 +731,29 @@ func init() {
 		}
 	}
 
+	if config.OTLP.Traces.Endpoint != "" {
+		var err error
+		otlpClient, err = outputs.NewClient("OTLP", config.OTLP.Traces.Endpoint, false, false, config, stats, promStats, statsdClient, dogstatsdClient)
+		if err != nil {
+			config.OTLP.Traces.Endpoint = ""
+		} else {
+			outputs.EnabledOutputs = append(outputs.EnabledOutputs, "OTLP.Traces")
+			log.Printf("[INFO] : OTLP.Traces=%+v\n", config.OTLP.Traces)
+			otlpShutdown = otlpInit()
```

Comment:
should be done with the creation of the client

Reply:
yeah much better, will re-factor it together with `otlp_init.go` move to `outputs/`

Reply:
^- [3ac1ad4](https://github.com/falcosecurity/falcosidekick/pull/613/commits/3ac1ad465ef0b5cb8c4860b455d18c4167193b51)